dpcomp_gpu_runtime/lib/gpu_runtime.cpp

```diff
@@ -19,6 +19,7 @@
 #include <cstdint>
 #include <cstdio>
 #include <cstdlib>
+#include <iostream>
```

Comment:
Including iostream comes with compile time and binary size costs. It is fine for examples but please do not use in runtime or compiler.